libraries/AC_PrecLand/AC_PrecLand.h

```diff
@@ -1,6 +1,13 @@
 #pragma once
 
 #include <AP_Math/AP_Math.h>
+
+#ifndef AP_PREC_LAND_ENABLED
+ #define AP_PREC_LAND_ENABLED 1
+#endif
```

Comment:
```suggestion

#include "AC_PrecLand_config.h"
```

please create a new file to hold this new define - you can see the pattern increasingly used throughout ArduPilot.

Also, please move the include of AP_Math.h to within the `#if` :-)

Reply:
I moved it out because of compilation errors (a "redefined define" error). I'll move it inside again and let's check